examples/04_manipulating_images/plot_roi_extraction.py

```diff
@@ -61,8 +61,9 @@
 
 # Second, load the labels stored in a text file into array using numpy
```

Comment:
using numpy -> using pandas. Also, can you please import numpy at relevant section ?